packages/story-editor/src/components/panels/design/layer/layerPanel.js

```diff
@@ -56,6 +52,14 @@ const Divider = styled.div`
 function LayerPanel() {
   const layers = useLayers();
 
+  const {
+    state: { inspectorContentHeight },
+  } = useInspector();
+
+  // We want the max height to fill the space underneath the document/design tab bar.
+  // Since the document/design tab bar is 50px subtract that from the full height of the inspector.
+  const maxHeight = inspectorContentHeight - 50;
```

Comment:
🤔   i'm trying to think there's a way we can pull in the height of the tab bar instead of assigning it by force here - just for when the designs update. It's set in components/tabview > `TabElement` and it's a combo of 32px height and 19px of vertical margin.

Reply:
I could probably calculate it via `useInspector` -> `tabRef` in the `InspectorProvider` . I'll. look into it.

Reply:
I think Brittany's thinking something much simpler. just making like `TAB_HEIGHT` & `TAB_MARGIN_VERTICAL` constants and using them both here and in the `TabElement`.

Reply:
thanks, max - yup! keep it simple 😎

Reply:
oh duh! Cool